Userland/DevTools/UserspaceEmulator/SoftCPU.h

```diff
@@ -375,6 +375,8 @@ class SoftCPU final
             return write_memory32(address, data);
     }
 
+    // why is it done like that and not with an enum?
+    // then maybe as a if constexpr?
```

Comment:
Please don't add random questions to the code unless they are part of a `FIXME` or `TODO`.
If you have questions about how something works, ask somebody :)

Reply:
Ok I will remove that question for now, may fix it when I come to it later down the line